src/ethereum/EthereumEngine.ts

```diff
@@ -1062,15 +1062,20 @@ export class EthereumEngine extends CurrencyEngine<
       this.makeSpendCheck(edgeSpendInfoIn)
     const { memos = [] } = edgeSpendInfo
 
-    const { pendingTxs = [], tokenId } = edgeSpendInfo
+    const {
+      pendingTxs = [],
+      tokenId,
+      allowChainedPending = false
+    } = edgeSpendInfo
     const unconfirmedTxs = this.getUnconfirmedTxs()
 
     // If we have pending transactions, that are not in the pendingTxs array,
-    // then throw an error:
+    // then throw an error UNLESS allowChainedPending is true:
     const unexpectedUnconfirmedTxs = unconfirmedTxs.filter(
       tx => !pendingTxs.some(ptx => ptx.txid === tx.txid)
     )
-    if (unexpectedUnconfirmedTxs.length > 0) {
+    
+    if (unexpectedUnconfirmedTxs.length > 0 && !allowChainedPending) {
```

Comment:
Instead of `allowChainedPending`, let's do `spendTarget.otherParams?.nonce != null`